server/src/controllers/v1/user.js

```diff
@@ -0,0 +1,92 @@
+import user from '../../models/user';
+import destructureData from '../utils/index';
+import { newToken } from '../utils/jwtToken';
+
+const health = (req, res) => {
+  res.status(200).json({ status: 'ok' });
+};
+
+const signinController = async (req, res) => {
+  try {
+    const user = await user.findOne({
+      email: req.body.email,
+    });
+    const isMatch = await user.checkPassword(req.body.password);
+    if (isMatch) {
+      const userData = destructureData(user);
+      const token = newToken(userData);
+      return res.status(200).json({
+        success: true,
+        message: 'Sign-in successful',
+        user: { ...userData, token },
+      });
+    }
+    return res.status(400).json({
+      sucess: false,
+      message: 'Sign-in failed. Check email and password',
+    });
+  } catch (error) {
+    return res.status(400).json({
+      sucess: false,
+      message: 'Email not found. Please sign up',
+    });
+  }
+};
+
+const signupController = async (req, res) => {
+  try {
```

Comment:
add some line breaks, the code is not readable

Reply:
Removed.